src/templates/tournaments/tournament_create.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Create/Update Tournament</h1>
```

Comment:
I would suggest changing this to just "Create Tournament" since this PR is not allowing us to edit any tournament information.